requirements.txt

```diff
@@ -1,10 +1,12 @@
-ansible>=2.0.0
+pip>=8.0.0
```

Comment:
I don't think this means anything. AFAIK, the new pip will be active only after all packages were installed.
Let's try it without this line